### PR TITLE
Feature/triangle hit

### PR DESCRIPTION
**Pull request by DanK2029**

### CODE REVIEW

File: test/scene/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+add_library (PathTracer.SceneTest OBJECT
+    triangle.cpp
```
Comment (farzonl) [inline test/scene/CMakeLists.txt:2]: This should be triangleTest.cpp

**Comment by farzonl** [comment]
In `src/CMakeLists.txt` you need to add at line 56 in `add_library` add `$<TARGET_OBJECTS:PathTracer.Scene>`
In the same file at line 69 add `add_subdirectory (scene)`

**Comment by farzonl** [comment]
in `test/CMakeLists.txt` you need to add $<TARGET_OBJECTS:PathTracer.SceneTest> to line 12 in `add_executable`
and at line 17 `add_subdirectory (scene)`

**Review by farzonl — CHANGES_REQUESTED** [review]
let me know when you are ready for a second review.

**Comment by DanK2029** [comment]
made the suggested fixes

**Review by farzonl — APPROVED** [review]

**Comment by farzonl** [comment]
LGTM

**Pull request merged**